db/migrate/20150608190453_create_results.rb

```diff
@@ -0,0 +1,17 @@
+class CreateResults < ActiveRecord::Migration
+  def change
+    create_table :results do |t|
+      t.integer :assessment_id
+      t.string :assessment_type
+      t.string :name
+      t.string :description
+      t.string :problem_description
```

Comment:
Yeah, I struggled with this a bit myself. These are the fields currently in the `direct_assessments` table that need to be editable by TA if, for example, the problem number for the assessment changes. `name` and `description` have different prefixes in the `direct_assessments` and `indirect_assessments` models (`assignment_` and `assessment_`, respectively). Would it be better to add the `assessment` prefix to those fields?

`problem_description` is only in the `direct_assessments` table, and I think it just adds more detail about the assessment.